packages/office-ui-fabric-react/src/utilities/color/interfaces.ts

```diff
@@ -1,37 +1,40 @@
-/** RGB color with optional alpha value. */
-export interface IRGB {
-  /** Red, range 0-255. */
-  r: number;
-  /** Green, range 0-255. */
-  g: number;
-  /** Blue, range 0-255. */
-  b: number;
-  /** Alpha, range 0 (transparent)-100. Usually assumed to be 100 if not specified. */
-  a?: number;
-}
-
-export interface IHSV {
-  /** Hue, range 0-359. */
-  h: number;
-  /** Saturation, range 0-100. */
-  s: number;
-  /** Value, range 0-100. */
-  v: number;
-}
-
-export interface IHSL {
-  /** Hue, range 0-359. */
-  h: number;
-  /** Saturation, range 0-100. */
-  s: number;
-  /** Lightness, range 0-100. */
-  l: number;
-}
-
-export interface IColor extends IRGB, IHSV {
-  /** Hex string for the color (excluding alpha component), *not* prefixed with #. */
-  hex: string;
-
-  /** CSS color string. If a hex value, it must be prefixed with #. */
-  str: string;
-}
+/** RGB color with optional alpha value. */
+export interface IRGB {
+  /** Red, range 0-255. */
+  r: number;
+  /** Green, range 0-255. */
+  g: number;
+  /** Blue, range 0-255. */
+  b: number;
+  /** Alpha, range 0 (transparent)-100. Usually assumed to be 100 if not specified. */
+  a?: number;
+}
+
+export interface IHSV {
+  /** Hue, range 0-359. */
+  h: number;
+  /** Saturation, range 0-100. */
+  s: number;
+  /** Value, range 0-100. */
+  v: number;
+}
+
+export interface IHSL {
+  /** Hue, range 0-359. */
+  h: number;
+  /** Saturation, range 0-100. */
+  s: number;
+  /** Lightness, range 0-100. */
+  l: number;
+}
+
+export interface IColor extends IRGB, IHSV {
+  /** Hex string for the color (excluding alpha component), *not* prefixed with #. */
+  hex: string;
+
+  /** CSS color string. If a hex value, it must be prefixed with #. */
+  str: string;
+
+  /* transparency*/

```

Comment:
I would prefer not to have a separate value in IColor for transparency because it makes a lot of internal logic unnecessarily complicated. Instead, just use the alpha value and if the consumer wants to display it as transparency they can do the conversion themselves. Or we can add a helper utility `getTransparency(color: Pick<IRGB, 'a'>)` .

Also in general, you need to use `/** doc comments */` (with two leading `**`) to make this show up properly in the documentation, and be more descriptive with the comment.

Reply:
Never mind, I see this is removed now.